test/bats/test.bats

```diff
@@ -6,16 +6,16 @@ WAIT_TIME=120
 SLEEP_TIME=1
 
 @test "osm pod is ready" {
-    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl wait --for=condition=Ready --timeout=60s pod -l app=osm-controller"
+    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl wait --for=condition=Ready --timeout=60s pod -l app=osm-controller -n arc-osm-system"
     assert_success
 }
 
 @test "configmap has been deployed" {
-    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get configmap/osm-config"
+    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get configmap/osm-config -n arc-osm-system"
     assert_success
 }
 
 @test "mutating webhook has been deployed" {
-    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get mutatingwebhookconfigurations.admissionregistration.k8s.io osm-webhook-osm"
+    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get mutatingwebhookconfigurations.admissionregistration.k8s.io arc-osm-webhook-osm -n arc-osm-system"
```

Comment:
mutatingwebhookconfigurations is a cluster scope resource, you don't need to pass in namespace value here.